gateway/gateway.ino

```diff
@@ -71,6 +73,21 @@ void loop() {
   kitchenTable.tick();
   workshop.tick();
   corridor.tick();
+  doorbell.tick();
+
+  switch (dzwonek) {
```

Comment:
1. Switch can be encapsulated in function (ie: ‘updateDoorbellState’) to avoid too long implementation of ‘loop’ and additionally it will automatically explain it’s purpose.  
2. As the current code doesn’t have any validation in any other places, it’s good practice to add implementation of default behaviour, even if it will never fall into it and there will be only ‘break’ instruction.